lens/util/repo.go

```diff
@@ -185,7 +185,8 @@ func MethodAndParamsForMessage(m *types.Message, destCode cid.Cid) (string, stri
 
 	params, method, err := ParseParams(m.Params, m.Method, destCode)
 	if method == "Unknown" {
-		return "", "", fmt.Errorf("unknown method for actor type %s: %d", destCode.String(), int64(m.Method))
```

Comment:
I'm not sure this is the correct behavior if a method is unknown. I don't think we can write empty strings to models that expect a method name. One option would be to remove returning the name of the method being called here, and instead post process tables needing a method name string. cc @birdychang

Reply:
Why not just put ‘unknown’ as the method name?

Reply:
We can always post process later. Also, don’t we wanna deprecate parsed messages? I don’t really care about the model. Let’s make it work first.

Reply:
Or just store the method code as the method name here. There’s should be separate columns in case it fails to lookup the name.

Reply:
Store the method in string format.